components/os-installer.sh

```diff
@@ -180,12 +180,15 @@ cp -a /mnt/sbin/fsck.ext2 /tmp/utils/
 cp -a /mnt/sbin/fsck.ext4 /tmp/utils/
 echo "Done."
 
+TUNE2FS=`which tune2fs`
```

Comment:
Will it return the path to the onie image's tune2fs utility or one which is installed on the Debian image?

Reply:
It will search in PATH :)